meson.build

```diff
@@ -120,6 +120,12 @@ gio = dependency('gio-2.0', version : '>= 2.45.8')
 giounix = dependency('gio-unix-2.0', version : '>= 2.45.8', required: false)
 lzma = dependency('liblzma')
 zstd = dependency('libzstd')
+if get_option('zstd')
+    if not zstd.found() 
+        error('libzstd is missing, please install it to continue!')
+    endif
+      conf.set('HAVE_ZSTD', 1)
```

Comment:
i think the indenting is wonky here -- two less spaces

Reply:
https://github.com/hughsie/libxmlb/blob/main/meson.build#L144 I wrote like this

Reply:
https://github.com/hughsie/libxmlb/pull/134/commits/a074f2e283c2333a317f3fbb7fad4d1677fea4da fixed.